pkg/cli/admin/upgrade/status/examples/4.15.0-ec2-unavailable-mco-20m.detailed-output

```diff
@@ -1,10 +1,15 @@
 = Control Plane =
 Assessment:      Progressing - Slow
 Target Version:  4.15.0-ec.2 (from 4.14.0-rc.3)
+Updating:        machine-config
 Completion:      97% (32 operators updated, 1 updating, 0 waiting)
 Duration:        59m (Est. Time Remaining: <10m)
 Operator Health: 30 Healthy, 2 Unavailable, 1 Available but degraded
 
+Updating Cluster Operators
+NAME             SINCE                  REASON   MESSAGE
+machine-config   2023-11-24T15:27:08Z            Working towards 4.15.0-ec.2
```

Comment:
Done.

My goal is to mimic the age. Also tried to find its implementation (so that we do not need to reinvent the wheel) but failed.

```console
oc --kubeconfig ~/.kube/ota-int.config get ns
NAME                                               STATUS        AGE
app-sre                                            Active        540d
cert-manager                                       Active        541d
cert-manager-operator                              Active        544d
cincinnati-integration                             Active        564d
dedicated-admin                                    Active        3y61d
default                                            Active        3y61d
eir                                                Active        3y59d
gersemi                                            Active        3y59d
graph-data-channel-stabilization                   Active        239d
havi                                               Active        3y59d
hive                                               Active        2y243d
homelab                                            Active        2y243d
jupyterlab                                         Active        2y353d
jupyterlab-operator-system                         Active        2y353d
kaas                                               Active        2y254d
keda                                               Active        25d
```

Reply:
Oh, now I actually saw the code, you're trying to make days and _years_ work. Is that worth the trouble? We're focusing on updates which ideally take hours, possibly several days at most (but then it is still acceptable to show hours).